### DISCUSSION TITLE
Add database schema values to write airflow metadata in values.yaml

### DISCUSSION THREAD

**Discussion by kimminw00** [apache/airflow#32421] [Ideas] [answered]
PostgreSQL 15 removes the global write privilege from the public schema.
So, it is required to specify which schema to write airflow metadata in values.yaml

Ref
https://www.crunchydata.com/blog/be-ready-public-schema-changes-in-postgres-15

**Comment by potiuk** [comment, marked as answer]
Do you want to make a PR with documentation update for that @JoeyValentine ? This is as easy as finding the right page - likely here: https://airflow.apache.org/docs/apache-airflow/stable/howto/set-up-database.html#setting-up-a-postgresql-database ?

It's as easy as typing "Suggest a change on this page" and PR will be open for you and you will be able to add appropriate note straight there without leavint Airflow UI.

Do you want to update it and become one of the > 2500 contributors to Airflow this way.
